devlab/jenkins/prep_filter.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+tenant_name="tenant1"
```

Comment:
What if `tenant1` is not available? Let's pass tenant name as argument to the script